cmd/login.go

```diff
@@ -25,6 +26,7 @@ import (
 
 // Login starts the login handshake with GitHub and okteto
 func Login() *cobra.Command {
+	log.Warning("'okteto login' will soon be depreciated, run 'okteto context' to configure your cluster.")
```

Comment:
okteto context configures the access to the cluster, not the actual cluster. Suggestion below. 

```suggestion
	log.Warning("'okteto login' will soon be deprecated in favor of 'okteto context'.")
```